javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -207,6 +211,12 @@ export async function generateNetworkSpec(
           nodeGroup.resources || networkSpec.relaychain.defaultResources,
         db_snapshot: nodeGroup.db_snapshot,
       };
+      if (nodeGroup.prometheus_prefix) {
+        node.prometheus_prefix = nodeGroup.prometheus_prefix;
+      } else if (config.relaychain.prometheus_prefix) {
+        node.prometheus_prefix = config.relaychain.prometheus_prefix;
+      }
+
```

Comment:
How about this ?

```suggestion
        node.prometheus_prefix = nodeGroup.prometheus_prefix || config.relaychain.prometheus_prefix;
```

Reply:
not really; cause there is  a chance that none of them exist, and in that case I want to avoid adding the key "prometheus_prefix" to the object